### DISCUSSION TITLE
spotbugs-gradle-plugin not released since 4.7.1 (2021-04-30)

### DISCUSSION THREAD

**Discussion by callistan** [spotbugs/spotbugs-gradle-plugin#717] [General]
Is there anything preventing releases since 5.x? 

https://repo.gradle.org/ui/packages/gav:%2F%2Fcom.github.spotbugs:spotbugs-gradle-plugin
https://mvnrepository.com/artifact/gradle.plugin.com.github.spotbugs.snom/spotbugs-gradle-plugin

```
> Task :buildSrc:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':buildSrc:compileJava'.
> Could not resolve all files for configuration ':buildSrc:compileClasspath'.
   > Could not find gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.6.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/gradle/plugin/com/github/spotbugs/snom/spotbugs-gradle-plugin/5.0.6/spotbugs-gradle-plugin-5.0.6.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :buildSrc
```

**Comment by callistan** [comment]
...and just learned about https://jitpack.io/#spotbugs/spotbugs-gradle-plugin/5.0.6
Can this be added to the README?

**Reply by KengoTODA** [reply]
I don't recommend using unofficial repositories like jitpack.
Just follow the interaction described in the official repository then it should be enough.  
https://plugins.gradle.org/plugin/com.github.spotbugs

If it does not work, make sure you're using the latest Gradle.
Refer to https://blog.gradle.org/jcenter-shutdown for detail.

**Comment by hazendaz** [comment]
closing as not the issue any more.  we are releasing regularly again.
